### DISCUSSION TITLE
Featured Products Properties

### DISCUSSION THREAD

**Discussion by MatthewJRyan** [openstore-ecommerce/OpenStore#158] [Q&A] [answered]
How are products set as Featured
I believe a property has to be created and then assigned on the product?
But how is this setup exactly?
the manager manual is in french and doesnt go into much detail or give an example of setting this up,

**Comment by DNNMonster** [comment, marked as answer]
Welcome to your new international family!  I'm still working on my French.

Once you have your product data set up in the Back Office you can just drop a new OS_ProductView module on the page like you would for normal product displays.

Then you should be able to set the module's List Template to the FeaturedCarousel.cshtml or FeaturedCycle.cshtml from the OS_ProductView settings. 

You will probably also want to adjust the OS_ProductView module's settings for you're featured product display to limit the products rendered to a specific property or category.

That should be all there is to it.
